src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ *
+ * @see Breadcrumb
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  /**
+   * Handles onClick event. If onClick is not defined, does nothing.
+   * */
+  const handleClick = (e) => {
+    if (!_.isUndefined(e)) e.preventDefault()
+    if (!_.isFunction(onClick)) return
+
+    onClick()
+  }
```

Comment:
Few notes on event handlers.

1. Never preventDefault (the user can prevent default if they want)
2. Always pass back the event
3. propTypes validates the onClick as being a function for us, so we don't have to include validation

The refactored `handleClick` would look like this:

```js
const handleClick = (e) => {
  if (onClick) onClick(e)
}
```

Since now all this function does is pass the event from one function to another, it can be removed  entirely.  We can just spread the `onClick` prop on the returned component.  When called, it will also be called with the event.

There are times that we want to first capture a click handler.  This would be when we want to pass some additional arguments after the event argument.